### PR TITLE
Portfolio accessibility

### PR DESCRIPTION
**Pull request by PraveenKumar141095**
**Added** ### 
> HTML file for both my mobile version and desktop as well.

> Added CSS files to give styling of them

> Added LINTER to find bugs in files.***

### CODE REVIEW

**Comment by EngruugbeJimmy** [comment]
**[Page titles]
-No issue found

[Image text alternatives** ]
-No issue found

[Text headings]
-No issue found

[Color contrast]
-No issue found

[Resize]
-No issue found

[Interaction]
-form label issue found, please include form labels.
Labels are critical for blind users, user with low vision, users with mobility disabilities and users with memory loss. Missing labels will make a form inaccessible for many users. 
Please click the link to the HTML code (https://github.com/Praveen1995-10-14/praveen_portfolio/blob/6db2d7080db3c6636779f47f4605fc1b8c2da921/index.html#L244-L248:~:text=%3Cinput%20type,%3C/textarea%3E)




[Moving content]
-No issue found

[Multimedia]
-No issue found

[The basic structure of the page]
-No issue found**

File: index.html [hunk 1]
```diff
@@ -5,63 +5,259 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Please make sure your website has a good page title. Take a look at [this article](https://www.w3.org/WAI/test-evaluate/preliminary/#titlehttps://www.w3.org/WAI/test-evaluate/preliminary/#title) for more information about it.
- It adequately and briefly describes the content of the page. For example:
 - Poor titles:
   - Microverse project | Your portfolio.
   - Portifolio project
 - Better page titles:
   - Mark's portfolio
   - Mark full-stack webdeveloper

File: index.html [hunk 2]
```diff
@@ -5,63 +5,259 @@
...
+                        <li>
+                            <a class="Twitter" href="">
+                                <img src="images/header/twiter.png" alt="">
+                            </a>
+                        </li>
+                        <li>
+                            <a class="M" href="">
+                                <img src="images/header/m.png" alt="">
+                            </a>
+                        </li>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Please, check if all the `img` tags have an `alt` attribute with some meaningful text describing the images.

File: index.html [hunk 3]
```diff
@@ -5,63 +5,259 @@
...
+                </div>
+                <div class="framework">
+                    <h1>Framework</h1>
+                    <span class="material-symbols-outlined">
+                        chevron_right
+                    </span>
+                </div>
+                <div class="line"></div>
+                <div class="skills">
+                    <h1>Skills</h1>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Kindly take a look at [this article](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) and make sure that the heading structure is following the correct hierarchy. The `h1` tag should be the main heading of the page and the page should not have more than one `h1` tag.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Praveen1995-10-14, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Professional PR message. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] The accessibility report is missing some issues found in the project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @Praveen1995-10-14, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

AlmostThereReachGIF


 ## Highlights


- The page is now accessible ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

 ## Required Changes ♻️

- [ ] As mentioned by the previous reviewers it's required to have the accessibility report, Please make sure to add the accessibility report in your pull request summary.

image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by EngruugbeJimmy** [comment]
# In this web accessibility project milestone:

*[Page titles]
-No issue found

[Image text alternatives** ]
-No issue found

[Text headings]
-No issue found

[Color contrast]
-No issue found

[Resize]
-No issue found

[Interaction]
-No issue found; form label issue found

[Moving content]
-No issue found

[Multimedia]
-No issue found

[The basic structure of the page]
-No issue found**

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## @Praveen1995-10-14  , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**